validator/keymanager/remote-web3signer/v1/custom_mappers.go

```diff
@@ -57,8 +57,8 @@ func MapAttestation(attestation *ethpb.Attestation) (*Attestation, error) {
 		return nil, err
 	}
 	return &Attestation{
```

Comment:
Please add regression tests for removal of `.Bytes()` as this is fundamentally very different data.